include/PoissonSubsampling.h

```diff
@@ -1,6 +1,25 @@
 #pragma once
 #include "SamplingMask.h"
 
+/**
+ * Samples an elliptical region according to a Poisson distribution
+ * @param lLines
```

Comment:
@spam-depository Maybe we can add parameter docs here and in the python data-class?